### DISCUSSION TITLE
Redeploy webapp if global admin completed initial deployment

### DISCUSSION THREAD

**Discussion by lachlan-c19** [microsoft/PubSec-Info-Assistant#721] [Q&A]
I'm exploring options to retrieve and redeploy existing configurations using GitHub Codespaces. I understand setting up a DevOps pipeline is the standard approach, are there any alternative methods available? Specifically, that would allow redeployment via a codespace using terraform? I only need to redeploy the web app UI, which was initially deployed by a global admin.

**Comment by dayland** [comment]
There are features coming in v1.1.1 that include migrating from v1.0 to v.1.1. This includes switch from BICEP to Terraform that has features:
- `make  import-state` for importing terraform state from and existing RG 
- `make prep-env` which assign roles to an RG you did not initially deploy

**Reply by ymac3** [reply]
Hi @dayland , what if the RG was not created by terraform?
